src/node_watchdog.h

```diff
@@ -37,16 +37,14 @@ namespace node {
 
 class Watchdog {
  public:
-  explicit Watchdog(v8::Isolate* isolate, uint64_t ms);
+  explicit Watchdog(
+    v8::Isolate* isolate,
+    uint64_t ms,
+    bool* timed_out = NULL);
```

Comment:
Ditto for 4-space indents. I think here we’d usually keep the `v8::Isolate* isolate` bit directly after the `(` and align all other parameters vertically below it; but again, this is just a tiny nit and can be fixed up while landing.